src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -132,8 +132,8 @@ export class UserFormComponent implements OnInit {
       () => {
         this.notificationService.show(
           NotificationType.success,
-          `User "${userFormModel.username}" has been created.`,
-          'Create User'
+          'Create user "${userFormModel.username}"',
+          `User "${userFormModel.username}" has been created.`
```

Comment:
This argument is now only used for error message descriptions. Please remove this line.